domains/anshalplayz.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/AnshalPlayz/project",
+    "owner": {
+        "username": "AnshalPlayz",
+        "email": "<email@address>",
+        "twitter": "<twitter-username>"
```

Comment:
Please fill out the email and twitter line (if you dont want to give out your email then leave it blank and just fill out the twitter line)